design/APISamples.md

```diff
@@ -0,0 +1,952 @@
+Message: There is not enough data available to generate a recommendation._**
+
+```
+{
+    "cluster_name": "cluster-one-division-bell",
+    "kubernetes_objects": [
+        {
+            "type": "deployment",
+            "name": "my-sample-deployment",
+            "namespace": "default",
+            "containers": [
+                {
+                    "container_image_name": "my-sample-image",
+                    "container_name": "my-sample-container",
+                    "recommendations": {
+                        "notifications": {
+                            "120001": {
+                                "type": "info",
+                                "message": "There is not enough data available to generate a recommendation.",
+                                "code": 120001
+                            }
+                        },
+                        "data": {}
+                    }
+                }
+            ]
+        }
+    ],
+    "version": "1.0",
+    "experiment_name": "gorgeous-dinakar"
+}
+```
+
+**_Code: 323001
+Message: CPU usage is less than one millicore, No CPU Recommendations can be generated._**
+
+```
+{
+    "cluster_name": "cluster-one-division-bell",
+    "kubernetes_objects": [
+        {
+            "type": "deployment",
+            "name": "my-sample-deployment",
+            "namespace": "default",
+            "containers": [
+                {
+                    "container_image_name": "my-sample-image",
+                    "container_name": "my-sample-container",
+                    "recommendations": {
+                        "notifications": {
+                            "112101": {
+                                "type": "info",
+                                "message": "Duration Based Recommendations Available",
+                                "code": 112101
+                            }
+                        },
+                        "data": {
+                            "2023-06-15T01:00:26.200Z": {
+                                "duration_based": {
+                                    "short_term": {
+                                        "monitoring_start_time": "2023-06-14T01:00:26.200Z",
+                                        "monitoring_end_time": "2023-06-15T01:00:26.200Z",
+                                        "duration_in_hours": 24.0,
+                                        "pods_count": 1,
+                                        "confidence_level": 0.0,
+                                        "current": {
+                                            "requests": {
+                                                "memory": {
+                                                    "amount": 646.35,
+                                                    "format": "MiB"
+                                                },
+                                                "cpu": {
+                                                    "amount": 1.432e-05
```

Comment:
1. Dont know what kind of input data that you are using, kubernetes will reject values like this as they are below a millicore. Please provide examples which are practical.
2. `format` key is missing
Same for several values in this file